src-ghc/Pact/GasModel.hs

```diff
@@ -81,25 +83,30 @@ csvHeader :: (T.Text, T.Text, T.Text, T.Text, T.Text, T.Text)
 csvHeader = ("function", "backendType", "description", "mean1", "mean2", "mean3")
 
 bench
-  :: T.Text
+  :: Bool
+  -> T.Text
   -> T.Text
   -> GasTest e
   -> IO BenchResults
-bench funName backendType test = do
+bench justBench funName backendType test = do
   terms <- compileCode (_gasTestExpression test)
   bracket setup teardown $ \s -> do
     putStrLn $ T.unpack funName ++ "/" ++
                T.unpack backendType ++ "/" ++
                T.unpack (_gasTestDescription test)
     r1 <- C.benchmark' (run terms s)
-    r2 <- C.benchmark' (run terms s)
-    r3 <- C.benchmark' (run terms s)
-    return (funName,
-            backendType,
-            (_gasTestDescription test),
-            (estPoint $ C.anMean $ C.reportAnalysis r1),
-            (estPoint $ C.anMean $ C.reportAnalysis r2),
-            (estPoint $ C.anMean $ C.reportAnalysis r3))
+    if justBench
+      then do
+        return (funName,backendType,(_gasTestDescription test),0,0,0)
```

Comment:
How about creating another function that handles just the `just benchmark` logic? As is, the when `justBench` is enabled, we're accumulating a list of `BenchResults` that's not used.